javascript-if/main.js

```diff
@@ -0,0 +1,92 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+function isUnderFive(number) {
+  let result;
+  if (number < 5) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isEven(number) {
+  let result;
+  if (number % 2 === 0) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function startsWithJ(string) {
+  let result;
+  if (string.charAt(0) === 'J') {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrink(person) {
+  let result;
+  if (person.age < 21) {
+    result = false;
+  } else {
+    result = true;
+  }
+  return result;
+}
+
+function isOldEnoughToDrive(person) {
+  let result;
+  if (person.age > 16) {
+    result = true;
+  } else {
+    result = false;
+  }
+  return result;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  let result;
+  if (person.age < 999) {
+    result = false;
+  }
+  return result;
+}
```

Comment:
What about if they're older than that?

---

![image](https://user-images.githubusercontent.com/16143771/152267739-68b1b91f-737c-47bb-9190-086f70e723b2.png)